app/views/filters/new.html.haml

```diff
@@ -1,8 +1,8 @@
 - content_for :page_title do
   = t('filters.new.title')
 
-= simple_form_for @filter, url: filters_path do |f|
-  = render 'filter_fields', f: f
+= simple_form_for @filter, url: filters_path do |form|
+  = render form
```

Comment:
It might be weird to anthropomorphize a local variable, but when I think of myself as a form here, it must be so satisfying to just get passed to a method as an argument and have the whole world around you know to just grab a partial named after YOU to render ... you!